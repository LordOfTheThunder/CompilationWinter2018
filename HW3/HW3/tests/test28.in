void IntFunc(int x) {
	int t;
}
void BoolFunc(bool x) {
	int t;
}
void ByteFunc(byte x) {
	int t;
}
void TestAll() {
	IntFunc(33);
	BoolFunc(true);
	ByteFunc(33b);
	IntFunc(33b);
}




void main() { print("I hate this course"); }



