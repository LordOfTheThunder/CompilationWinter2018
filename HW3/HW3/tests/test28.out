---end scope---
x INT -1
t INT 0
---end scope---
x BOOL -1
t INT 0
---end scope---
x BYTE -1
t INT 0
---end scope---
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
IntFunc (INT)->VOID 3
BoolFunc (BOOL)->VOID 4
ByteFunc (BYTE)->VOID 5
TestAll ()->VOID 6
